src/main/resources/templates/db_scripts/all_finance_data_view.sql

```diff
@@ -17,20 +17,23 @@ SELECT
     'budgetName', budget.jsonb ->>'name',
     'budgetStatus', budget.jsonb ->>'budgetStatus',
     'budgetInitialAllocation', budget.jsonb ->>'initialAllocation',
+    'budgetCurrentAllocation', (budget.jsonb->'initialAllocation')::decimal + (budget.jsonb->'allocationTo')::decimal - (budget.jsonb->'allocationFrom')::decimal,
     'budgetAllowableExpenditure', budget.jsonb ->>'allowableExpenditure',
     'budgetAllowableEncumbrance', budget.jsonb ->>'allowableEncumbrance',
     'budgetAcqUnitIds', budget.jsonb ->'acqUnitIds',
     'groupId', groups.id,
     'groupCode', groups.jsonb ->> 'code'
   ) as jsonb
 FROM ${myuniversity}_${mymodule}.fiscal_year
-LEFT OUTER JOIN ${myuniversity}_${mymodule}.ledger
-    ON ledger.fiscalyearoneid = fiscal_year.id
-LEFT OUTER JOIN ${myuniversity}_${mymodule}.fund
+INNER JOIN ${myuniversity}_${mymodule}.fiscal_year fy_one
+    ON fy_one.jsonb->>'series' = fiscal_year.jsonb->>'series'
```

Comment:
gin index is created for series field and not used here, but amount of data for this table is not so big, probably we can leave as is